Makefile

```
@@ -54,10 +54,8 @@ whitespace-check: $(all_sources)
 vet: $(cmd_sources) $(pkg_sources)
 	go vet ./pkg/... ./cmd/...
 
-goimports-check: $(cmd_sources) $(pkg_sources)
-	go run ./vendor/golang.org/x/tools/cmd/goimports -d ./pkg ./cmd
-
-test: $(GINKGO)
+test:
+	 bash -c "[  -f ./build/_output/bin/ginkgo ] || GOBIN=$$PWD/build/_output/bin/  go install github.com/onsi/ginkgo/ginkgo"
```

`-f` doesn't seem like the Makefile way... better to do something like:

```suggestion
ginkgo:  ./build/_output/bin/ginkgo
	GOBIN=$$PWD/build/_output/bin/  go install github.com/onsi/ginkgo/ginkgo
test: ginkgo
```

---

done

---

Got a problem with ginkgo:  $$PWD/build/_output/bin/ginkgo